### PR TITLE
Incorporate pricing doc feedback

### PR DESCRIPTION
**Pull request by jessla**
cc @sksembhi @ehfeng for review and approval.

### CODE REVIEW

File: docs/pricing.rst [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
 ~~~~~~~~
 
-Capacity is the total number of events sent by and billed to your organization. Capacity comes in two types: on-demand and reserved.
+Capacity is the total number of events sent by and billed to your organization. You can purchase capacity in two ways: on-demand and reserved.
 
 On-demand Capacity
 ``````````````````
 
-On-demand capacity is billed at the end of each billing cycle. You are charged per processed event, rounded up to the nearest cent.
+On-demand capacity is billed at the end of each billing cycle. You are charged per processed event, rounded up to the nearest cent. You have the ability to set on-demand caps so you can stay on budget. See _`Billing Controls <https://docs.sentry.io/pricing/#billing-controls>`_ for more information.
```
Comment (jessla) [inline docs/pricing.rst]: @ehfeng is there a better way to link within the doc?

Comment (dcramer) [inline docs/pricing.rst]: With ``:ref:``:

http://www.sphinx-doc.org/en/1.5.1/markup/inline.html#cross-referencing-arbitrary-locations

And links something like this (but using whatever their docs say):

```
:doc:`Raven.js documentation <../../../clients/javascript/index>`
```

Comment (jessla) [inline docs/pricing.rst]: @dcramer updated at https://github.com/getsentry/sentry-docs/pull/89/files#diff-62c675fb84a7f06f516f729d4753f9a6R22

**Comment by jessla** [comment]
@ckj I don't have write access to this repo so I can't merge 😄

**Pull request merged**